prow/scripts/cluster-integration/serverless-integration-k3s.sh

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail  # Fail a pipe if any sub-command fails.
+
+date
+
+# https://github.com/kyma-project/test-infra/pull/2967 - explanation for that kaniko image
+export KANIKO_IMAGE="eu.gcr.io/kyma-project/external/aerfio/kaniko-executor:v1.3.0"
+export DOMAIN=${KYMA_DOMAIN:-local.kyma.dev}
+if [[ -z $REGISTRY_VALUES ]]; then
+  export REGISTRY_VALUES="dockerRegistry.enableInternal=false,dockerRegistry.serverAddress=registry.localhost:5000,dockerRegistry.registryAddress=registry.localhost:5000,containers.manager.envs.functionBuildExecutorImage.value=${KANIKO_IMAGE}"
+fi
+
+export KYMA_SOURCES_DIR="./kyma"
+
+host::create_registries_file(){
+cat > registries.yaml <<EOL
+mirrors:
+  registry.localhost:5000:
+    endpoint:
+    - http://registry.localhost:5000
+  configs: {}
+  auths: {}
+  
+EOL
+}
+
+host::create_coredns_template(){
+cat > coredns-patch.tpl <<EOL
+data:
+  Corefile: |
+    registry.localhost:53 {
+        hosts {
+            REGISTRY_IP registry.localhost
+        }
+    }
+    .:53 {
+        errors
+        health
+        ready
+        kubernetes cluster.local in-addr.arpa ip6.arpa {
+          pods insecure
+          upstream
+          fallthrough in-addr.arpa ip6.arpa
+        }
+        hosts /etc/coredns/NodeHosts {
+          reload 1s
+          fallthrough
+        }
+        prometheus :9153
+        forward . /etc/resolv.conf
+        cache 30
+        loop
+        reload
+        loadbalance
+    }
+EOL
+}
+
+host::update_etc_hosts(){
+  # needed for external docker registry
+  echo "${REGISTRY_IP} registry.localhost" >> /etc/hosts
+}
+
+host::create_docker_registry(){
+cat > registries.yaml <<EOL
+mirrors:
+  registry.localhost:5000:
+    endpoint:
+    - http://registry.localhost:5000
+  configs: {}
+  auths: {}
+  
+EOL
+
+mkdir -p /etc/rancher/k3s
+cp registries.yaml /etc/rancher/k3s
+
+docker run -d \
+  -p 5000:5000 \
+  --restart=always \
+  --name registry.localhost \
+  -v "$PWD/registry:/var/lib/registry" \
+  registry:2
+}
+
+install::prereq(){
+    curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+    apt-get -y install jq
```

Comment:
jq is already present in the `eu.gcr.io/kyma-project/test-infra/kyma-integration:v20200817-1.14.0-rc1-75-g625c1054-k8s1.16` image. You can drop this line

Reply:
never mind, this is executed on vm not in the docker container...